ambry-server/src/test/java/com.github.ambry.server/AmbryRequestsTest.java

```diff
@@ -518,6 +519,22 @@ public void listOfOriginalStoreKeysGetTest() throws Exception {
     }
```

Comment:
Minor: this method is supposed to throw `InterruptedException` and `IOException` to make it consistent with java doc.

Reply:
Done